### ISSUE TITLE
3 skipped on first race

### ISSUE THREAD

**Issue by PavanLuca** [flathub/com.agateau.PixelWheels#5]
When I start the first race of every game session it skips 3 suddenly.

**Comment by agateau** [comment]
Hi! Do you mean the `3` in the 3...2...1 count down? If so, that's odd. Do you have a way to record a video of the bug? (might be tricky since recording could affect the result)

**Comment by PavanLuca** [comment]
Yeah, it's on the countdown. I know about that thing about recordings, I tried OBS Studio and other screen capture tools before but all with scary frameskipping (not surprising knowing my laptop specs) so now I simply use my phone in such cases, I'll make you see that issue as soon as I can.
